packages/library/README.md

```diff
@@ -178,225 +178,119 @@ Here’s an overview of how to use the new library to interact with the RPC, con
 
 web3.js 2.0 ships with an implementation of the [JSON RPC specification](https://www.jsonrpc.org/specification) and a type spec for the [Solana JSON RPC](https://docs.solana.com/api).
 
-### Initializing a Default RPC API
+The main package responsible for managing communication with an RPC is `@solana/rpc`. However, this package makes use of more granular packages to break down the RPC logic into smaller pieces. Namely, these packages are:
 
-Here’s an example of creating the default API for interacting with the Solana JSON RPC:
+-   `@solana/rpc`: Contains all logic related to sending Solana RPC calls.
+-   `@solana/rpc-api`: Describes all Solana RPC methods using types.
+-   `@solana/rpc-transport-http`: Provides a concrete implementation of an RPC transport using HTTP requests.
+-   `@solana/rpc-spec`: Defines the JSON RPC spec for sending RPC calls using proxies.
```

Comment:
The proxy thing is sort of incidental. Nobody needs to know that, and also _part_ of the sender is concrete code (ie. the param/response transforms) so it's not _completely_ a proxy.